### ISSUE TITLE
How to use my private trained model in new training

### ISSUE THREAD

**Issue by amankumarjain** [ultralytics/hub#294]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

I have trained my basic model with dataset 1 and now I want to retrain on dataset2 by using the trained model, but while training setup i am not able to find my custom trained model in list of pre-existing models, pls let me know how to do that ?

**Comment by kalenmike** [comment]
@amankumarjain Thanks for asking the question. We have not yet enabled the option to train on top of an existing custom model. Currently we only support training from one of our official pre-trained models. We hope to release this feature in the near future.

**Comment by amankumarjain** [comment]
@kalenmike thank you for your reply

I am trying to preview my custom model on ultralystics android app which is also not working, my model is not visible in android application, am I missing something?

**Comment by kalenmike** [comment]
@amankumarjain Is your model attached to a project? Can you send me your model ID so I cant take a look and see if there are any issues?  The model ID is the last part of the URL when viewing the model in HUB:
`https://hub.ultralytics.com/models/XXXXXXXX`

**Comment by sergiossm** [comment]
@amankumarjain I just fixed the issue and submitted the new version, should be live in a few hours. Thanks for reporting!

**Comment by kalenmike** [comment]
@amankumarjain You will need to update to the latest version once it is deployed.

**Comment by amankumarjain** [comment]
Hi @kalenmike 
I have retried it today by reinstalling the app but still, my model `https://hub.ultralytics.com/models/2ocCZ75kJvVWfObkOQKH` is not visible in the ultralytics app

**Comment by glenn-jocher** [comment]
@amankumarjain Thank you for following up. In most cases, it may take a few minutes for model updates to appear in the app. Could you please check again and confirm if you still do not see your updated model in the app? If you still can't see the model, please let me know and I'll investigate further. Thank you!

**Comment by amankumarjain** [comment]
Hi @glenn-jocher I have reconfirmed it in the app, but I am not able to find models.
I have attached a screenshot, please check it and let me know if I am doing something wrong.

In the project screenshot, it shows 1 model but it's not visible to select for detection, if you need more details pls let me know.

![WhatsApp Image 2023-06-08 at 11 31 16](https://github.com/ultralytics/hub/assets/11255032/bc350246-1e6c-4049-a2d1-5388d3f200ab)
 
![WhatsApp Image 2023-06-08 at 11 31 16 (1)](https://github.com/ultralytics/hub/assets/11255032/5ba7a9c0-502f-4c36-8a48-193b77fc8f22)

**Comment by sergiossm** [comment]
@amankumarjain We'll let you know when the update is live. Thank you for your patience.

**Comment by amankumarjain** [comment]
Hi,
hub app is showing my model now, thanks for the quick resolution.

now it has a few other issues like
1. camera input is lagging a lot while detection
2. model precision was very high on validation n testing data but in camera input a lot of false detection.
3. on the medium architecture model fps was only 5fps which is very low, how to increase it ? (device camera has 30+fps)

**Comment by sergiossm** [comment]
@amankumarjain I'm glad to hear that you can now see your model. While we continue working on improving the performance of YOLOv8m, L, and X models on Android, I suggest considering training a YOLOv8n model instead. By transitioning to the YOLOv8n model, you can expect minimal accuracy loss while experiencing significant improvements in performance, specifically in terms of camera lag, on your mobile device.
